packages/manager/modules/billing/src/payment/method/add/controller.js

```diff
@@ -24,6 +25,7 @@ export default class BillingPaymentMethodAddCtrl {
     this.ovhPaymentMethodHelper = ovhPaymentMethodHelper;
     this.OVH_PAYMENT_METHOD_TYPE = OVH_PAYMENT_METHOD_TYPE;
     this.OVH_PAYMENT_METHOD_INTEGRATION_TYPE = OVH_PAYMENT_METHOD_INTEGRATION_TYPE;
+    this.coreConfig = coreConfig;
```

Comment:
**nitick:** `@ovh-ux/ng-ovh-payment-method` which is a public component shouldn't use `coreConfig` (from `@ovh-ux/manager-core`) which is a private package.

Reply:
Will be addressed in a separate pull request.